src/Storages/MergeTree/BackgroundProcessingPool.h

```diff
@@ -9,6 +9,8 @@
 #include <shared_mutex>
 #include <atomic>
 #include <functional>
+#include <pthread.h>
```

Comment:
Normally, you would want to avoid excessive inclusions as they WILL increase build time.